dotcom-rendering/src/web/components/YoutubeBlockComponent.importable.tsx

```diff
@@ -84,11 +85,16 @@ export const YoutubeBlockComponent = ({
 	width = 460,
 	duration,
 	origin,
+	stickyVideos,
 }: Props): JSX.Element => {
 	const [consentState, setConsentState] = useState<ConsentState | undefined>(
 		undefined,
 	);
 
+	// TODO: connect StickyVideo wrapper once
+	// feature flag is operational
+	console.log('Videos should stick', stickyVideos);
```

Comment:
https://trello.com/c/DO8DYFta/251-sticky-video-connect-stickyvideo-wrapper-component-to-youtubeblockcomponent-and-main-media